tests/ut/ir/parser/test_world_size.py

```
@@ -117,7 +117,7 @@ def find_for(stmt: ir.Stmt) -> ir.ForStmt | None:
 
 
 def test_world_size_rejects_positional_args():
-    with pytest.raises(Exception, match="no positional arguments"):
+    with pytest.raises(Exception, match="no positional arguments|takes 0 positional"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)